frontend/appflowy_flutter/lib/features/mension_person/data/models/invite.dart

```diff
@@ -0,0 +1,48 @@
+import 'package:flutter/material.dart';
+
+import 'person.dart';
+
+class InviteInfo {
+  InviteInfo({
+    required this.email,
+    this.role = PersonRole.member,
+    this.contactDetail,
+  });
+
+  final String email;
+  final PersonRole role;
```

Comment:
I added the `ShareRole` before, we can reuse it and rename it to `WorkspaceUserRole`.